### ISSUE TITLE
LEGO® Builder's Journey: Will not launch in DX12 mode.

### ISSUE THREAD

**Issue by Bitwolfies** [HansKristian-Work/vkd3d-proton#1243]
When launching from Steam, you're given 3 options, Launch as normal, "compatibility mode" (appends -dx11 to launch) and "low spec mode" (uses a separate exe and appends -dx11) yet launching normally always places the game into DX11 mode, my best guess is its unable to detect DX12 support, and thus forces compatibility mode. 

I suspect a workaround like the one created for Control may be needed. https://github.com/HansKristian-Work/vkd3d-proton/pull/1190

Thank you!

**Comment by K0bin** [comment]
Can you try it with a DXVK master build? It might check the feature level on a D3D11 device and DXVK has returned something <12 until very recently.

**Comment by Bitwolfies** [comment]
Tried it with GE 35, which was re based from master 3 days ago, apologies if that's not the same thing.

**Comment by doitsujin** [comment]
Can we please have a proton log?

**Comment by Bitwolfies** [comment]
Sure, here's the whole kit and caboodle. 

[This is the log from when its launched normally, so what should be DX12 mode, using the DXR11 config.](https://gist.github.com/Bitwolfies/0f5e76e8e6db33296ace4b3c20bb8ef7)

[This is the log from when its launched in compatibility mode, same exe just with -dx11 added, same DXR11 config.](https://gist.github.com/Bitwolfies/8cf8b3cb87398d99b88f9a52799753cb)

[Heres my hw dump from steam, but if you want the short, Arch, Kernel 15.19.9, NVIDIA driver 515.65 DMKS, GTX3080.](https://gist.github.com/Bitwolfies/f12db2c5d7a7c5d0a304d51a08f6ea7d)

**Comment by HansKristian-Work** [comment]
I tried it. It tries creating a device but it just exits right away.

With `PROTON_HIDE_NVIDIA_GPU=0 PROTON_ENABLE_NVAPI=1 VKD3D_CONFIG=dxr11 VKD3D_DEBUG=trace VKD3D_LOG_FILE=/tmp/lego.txt %command%`

[lego.txt](https://github.com/HansKristian-Work/vkd3d-proton/files/9634802/lego.txt)

It creates a device, then a command queue and from there it just releases everything. Dunno really.

**Comment by philipl** [comment]
Today's Proton Experimental update has enabled ray tracing. I can now see the options, turn them on, see a visual difference, and see my frame rate tank. I was able to turn on all the different options, and everything seems to work correctly. The Proton changelog doesn't note anything relevant for this release.

**Comment by Blisto91** [comment]
Yes the experimental changelogs usually does not include all of the upstream changes and fixes.

Work has been made so vkd3d-proton and dxvk together support D3D11On12 https://github.com/HansKristian-Work/vkd3d-proton/pull/1472
The only known problem right now afaik is that RT doesn't work on AMD RADV. But that is a driver issue.
